addons/mail/static/src/components/message_reaction_group_for_summary/message_reaction_group_for_summary.js

```diff
@@ -0,0 +1,24 @@
+/** @odoo-module **/
+
+const { Component } = owl;
+import { registerMessagingComponent } from '@mail/utils/messaging_component';
+
+export class MessageReactionGroupForSummary extends Component {
+
+    get messageReactionGroup() {
+        return this.messaging.models['mail.message_reaction_group'].get(this.props.messageReactionGroupLocalId);
+    }
```

Comment:
I'm not sure to understand what is this component, but it should have a corresponding model, not re-using an existing one